docs/Researcher/user-interface/trainings.md

```diff
@@ -28,23 +28,24 @@ To add a training:
 
 1. Press **Tranings** in the menu.
 2. In the *Projects* pane, select the destination project. Use the search box to find projects that are not listed. If you can't find the project, see your system administrator.
-3. In the *Templates* pane, select a template from the list. Use the search box to find templates that are not listed. If you can't find the specific template you need, see your system administrator.
-4. In the *Training name* pane, enter a name for the *Traninng*, then press continue.
-5. In the *Environment* pane select or [create a new environment](workspaces/create/create-env.md). Use the search box to find environments that are not listed.
-6. In the *Compute resource* pane, select resources for your tranings or [create a new compute resource](workspaces/create/create-compute.md). Use the search box to find resources that are not listed. Press *More settings* to use **Node Affinity** to limit the resources to a specific node.
-7. Open the *Volume* pane, and press *Volume* to add a volume to your training.
+3. In the *Multi-node* pane, choose `Single node` for a single node training, or `Multi-node (distributed)` for distributed training. When you choose `Multi-node`, you will then choose a framework that is listed. Then choose the `multi-node` training configuration by selecting either `Workers & master` or `Workers only`.
+4. In the *Templates* pane, select a template from the list. Use the search box to find templates that are not listed. If you can't find the specific template you need, see your system administrator.
+5. In the *Training name* pane, enter a name for the *Traninng*, then press continue.
+6. In the *Environment* pane select or [create a new environment](workspaces/create/create-env.md). Use the search box to find environments that are not listed. Press *More settings* to add an `Environment variable` or to edit the *Command* and *Arguments* field for the environment you selected.
```

Comment:
what do you mean by "use the search box to find environments that are not listed."

Reply:
In all of our grid forms there is now a dearch box that allows you to search for the asset of that specific grid. This is a standard statement that is in all of the docs, and any asset that has this kind of grid selection process has this line.

Reply:
there is a search box because the card he user is looking for may not be on the first page of the gallery. The term "not listed" is out of place IMO, WDYT?